tests/Unit/FluidXPlateTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
```suggestion
<?php declare(strict_types=1);
```

Tip: modify your PhpStorm templates